samples/config/petstore/protobuf-schema/services/store_service.proto

```diff
@@ -10,7 +10,7 @@
 
 syntax = "proto3";
 
-package petstore;
+package petstore.services.storeservice;
```

Comment:
Would "petstore.services.store_service" be a better package name?

If that's the case, then I think we should use the lambda.underscore instead of lambda.lowercase.

Reply:
To be honest I don't even like to put "class name" into the package. The official documentations says this: 

_Packages
Package name should be in lowercase, and should correspond to the directory hierarchy. e.g., if a file is in my/package/, then the package name should be my.package._

There is also an another way(?) to keep the original package name and then use some kind of prefix/postfix to the Request/Response message itself. I'm scared of ugly code generated by protoc tho.